rocky/rocky/templates/findings/finding_list.html

```diff
@@ -13,96 +13,125 @@
                 <h1>{% translate "Findings" %}</h1>
                 <p>
                     {% blocktranslate trimmed with observed_at|date as date %}
-        An overview of all findings OpenKAT found on {{ date }}. Each finding relates to an object.
-        Click a finding for additional information.
-      {% endblocktranslate %}
+                        An overview of all findings OpenKAT found on {{ date }}.
+                        Each finding relates to an object.
+                        Click a finding for additional information.
+                    {% endblocktranslate %}
                 </p>
                 {{ object_type_filters }}
                 {% include "partials/findings_list_toolbar.html" %}
 
                 {% translate "Object list" as filter_title %}
-                {% include "partials/ooi_list_filters.html" with title=filter_title %}
-
-                <div class="horizontal-scroll">
-                    <table class="action-buttons nowrap">
-                        <caption>{% translate "Showing " %}{{ object_list|length }} {% translate "of" %} {{ paginator.count }} {% translate "objects" %}</caption>
-                        <thead>
-                            <tr>
-                                <th>Nr</th>
-                                <th>{% translate "Severity" %}</th>
-                                <th>{% translate "Finding" %}</th>
-                                <th>{% translate "Tree" %}</th>
-                                <th>{% translate "Graph" %}</th>
-                                <th>{% translate "Details" %}</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            {% for hydrated_finding in object_list %}
+                {% if object_list|length > 1 and perms.tools.can_mute_findings %}
```

Comment:
This is a UX thing, by having from 2 items and more to make it a bulk action, otherwise one can just mute that single item at the object detail page.